app/component/navbar.tsx

```diff
@@ -126,10 +135,21 @@ function AccountSelector() {
     setIsAccountSelectorModalOpened(!isAccountSelectorModalOpened)
   return (
     <>
-      <div className="cursor-pointer" onClick={toggleAccountSelectorModal}>
-        <span>{address.ellipsize(account.address)}</span>
-        <FontAwesomeIcon icon={faCaretDown} className="ml-2" />
-      </div>
+      {/* Home page account selector button */}
+      <button
+        className="p-[7px_20px_7px_20px] flex items-center rounded-full border-[1px] border-solid border-black"
+        onClick={toggleAccountSelectorModal}>
+        <div className="mr-[12px] flex flex-col items-start">
+          <div className="leading-[19.4px] text-[16px] font-[Inter] font-[400] text-[#000000] whitespace-nowrap">
+            Jesse's wallet
+          </div>
+          <div className="leading-[14.6px] text-[12px] font-[Inter] font-[400] text-[#989898]">
```

Comment:
You can remove `font-[Inter]` and `font-[400]` since we added the default font style in this [PR](https://github.com/pilagod/waallet-extension/pull/147).